app/views/unattended/report_templates/host_-_installable_errata_count.erb

```diff
@@ -0,0 +1,38 @@
+<%#
+name: Host - Installable Errata Count
+snippet: false
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    for report on all available hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+  hidden_value: false
+model: ReportTemplate
+require:
+- plugin: katello
+￼ version: 4.9.0
+-%>
+<%- report_headers 'Host', 'Security Errata Count', 'Bugfix Errata Count', 'Enhancement Errata Count', 'Upgradable RPM Count' -%>
+<%- load_hosts(search: input('Hosts filter'), preload: [:content_facet]).each_record do |host| -%>
+<%- content_facet = host_content_facet(host) -%>
+<%-   
+   security_errata_count = content_facet.try(:installable_security_errata_count)
```

Comment:
This fails with `Safemode::NoMethodError: undefined method '#try' for Katello::Host::ContentFacet::Jail (Katello::Host::ContentFacet)`